### DISCUSSION TITLE
Trying to run high priority tasks without gaps.

### DISCUSSION THREAD

**Discussion by christianbrugger** [apache/airflow#26856] [Q&A]
I am having a a DAG which I want to finish as soon as possible once it is started.  

I am using the priority feature, but it doesn't seem to work.

What happens is that from time to time, tasks from other DAGs are schedule, although they have much lower priority. Those then block the slots of my pool and then there are gaps in the high priority DAG processing.

My DAG is always ready to be scheduled, and never on retry, so I don't understand this behavior. But it seems like there is a gap when one task is finished and before another task is started, where other DAGs are then allowed to run.

**Comment by potiuk** [comment]
Duplicate of https://github.com/apache/airflow/discussions/26821
